core/src/components/datetime/test/basic/datetime.e2e.ts

```diff
@@ -394,6 +394,60 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
   });
 });
 
+/**
+ * Synthetic IntersectionObserver fallback behavior.
+ *
+ * This test stubs IntersectionObserver so that the callback
+ * never reports an intersecting entry. The datetime should
+ * still become ready via its internal fallback logic.
+ */
+configs({ modes: ['md'], directions: ['ltr'] }).forEach(({ title, config }) => {
+  test.describe(title('datetime: IO fallback'), () => {
```

Comment:
The GH issue is missing. Here's an [example](https://github.com/ionic-team/ionic-framework/blob/2bebbd7a3e50c030f80e0b031d7f40d3ca13e92b/core/src/components/alert/test/a11y/alert.e2e.ts#L87) of what I mean.